configure.ac

```diff
@@ -272,9 +272,11 @@ AC_SUBST(QBT_REMOVE_CONFIG)
 AC_SUBST(QBT_ADD_DEFINES)
 AC_SUBST(QBT_REMOVE_DEFINES)
 
-AC_OUTPUT(conf.pri)
+AC_CONFIG_FILES(conf.pri)
+AC_OUTPUT
```

Comment:
Note that the big changes in `configure` script is caused by upgrading autoconf 2.69 -> 2.70, not by this. The change here doesn't really affect the generated `configure` script.